app.js

```diff
@@ -8,4 +8,133 @@ menu.addEventListener('click', () => {
 window.addEventListener('scroll', () => {
   menu.classList.remove('fa-times');
   navbar.classList.remove('nav-toggle');
+});
+
+// See project
+// document.getElementById('button').addEventListener('click', () => {
+//   document.querySelector('.bg-modal').style.display = 'flex';
+// });
+
+// document.querySelector('.close-button').addEventListener('click', () => {
+//   document.querySelector('.bg-modal').style.display = 'none';
+// });
+
+// document.getElementById('button-1').addEventListener('click', () => {
+//   document.querySelector('.bg-modal').style.display = 'flex';
+// });
+
+// document.getElementById('button-2').addEventListener('click', () => {
+//   document.querySelector('.bg-modal').style.display = 'flex';
+// });
+
+// document.getElementById('button-3').addEventListener('click', () => {
+//   document.querySelector('.bg-modal').style.display = 'flex';
+// });
```

Comment:
- [ ] As asked in the last point in [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) kindly remove the commented code

![image](https://user-images.githubusercontent.com/73140461/151215974-acb23223-3cef-4321-ae26-94d9ac331e96.png)